Frameworks/CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -498,6 +498,7 @@ CF_INLINE CFRange CFRangeMake(CFIndex loc, CFIndex len) {
 CF_EXPORT
 CFRange __CFRangeMake(CFIndex loc, CFIndex len);
 
+#define NSMakeRangeFromCF(cfr) NSMakeRange(cfr.location == kCFNotFound ? NSNotFound : cfr.location, cfr.length)
```

Comment:
Good point. Let me move this to NS*

---
In reply to: [77900982](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77900982) [](ancestors = 77900982)